test/ingestors/foundation.test.ts

```diff
@@ -0,0 +1,138 @@
+import { expect } from 'chai';
+import { FoundationIngestor } from '../../src/ingestors/foundation';
+import { mintIngestorResources } from '../../src/lib/resources';
+import { EVMMintInstructions } from '../../src/lib/types/mint-template';
+import { MintTemplateBuilder } from '../../src/lib/builder/mint-template-builder';
+
+describe('foundation', function () {
+  it('supportsUrl: Returns false for an unsupported URL', async function () {
```

Comment:
It'd be great to also inclue the `basicIngestorTests()` macro for some smoke tests -- these will be used in certain smoke test scenarios, as well as to add new general validity requirements in time. 🙏 

e.g. https://github.com/floornfts/mobile-minting/blob/main/test/ingestors/prohibition-daily.test.ts#L11